securedrop/tests/functional/test_journalist.py

```diff
@@ -59,14 +58,17 @@ def test_journalist_uses_index_delete_collections_button_modal(self):
         self._journalist_logs_in()
         self._journalist_uses_delete_collections_button_confirmation()
 
-    @pytest.mark.xfail(reason="Filter widget not displayed in TBB 9.0.1")
     def test_journalist_interface_ui_with_modal(self):
         self._source_visits_source_homepage()
         self._source_chooses_to_submit_documents()
         self._source_continues_to_submit_page()
         self._source_submits_a_file()
         self._source_logs_out()
 
+        # Toggle security slider to force prefs change
+        self.set_tbb_securitylevel(1)
```

Comment:
it's unclear where these numbers came from, can we define these numbers as constants somewhere and reference them here?

Reply:
Done.